src/amo/components/Addon/index.js

```diff
@@ -490,7 +490,7 @@ export class AddonBase extends React.Component {
                 authorNames={addon.authors.map((author) => author.username)}
                 className="Addon-MoreAddonsCard"
                 forAddonSlug={addon.slug}
-                numberOfAddons={3}
```

Comment:
Just out of interest, where did the `3` and `4` come from?

Reply:
The 4 was arbitrary but the three was for a single row. I didn't check prod and I think I tested against some add-ons that just happened to have three as a match locally so thought it was right. Oops.